Rubberduck.Parsing/Preprocessing/ConditionalCompilationConstantExpression.cs

```diff
@@ -29,8 +27,8 @@ public override IValue Evaluate()
             var constantValue = _expression.Evaluate();
             _symbolTable.Add(identifier, constantValue);
             return new LivelinessExpression(
-                isAlive: new ConstantExpression(new BoolValue(false)),
-                tokens: _tokens).Evaluate();
```

Comment:
Removing unnecessary argument names is not always good. They are usually there to aid readability.

Reply:
Will revert. Apologies for any overeager changes.